CHANGELOG.md

```diff
@@ -4,6 +4,385 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.11.0-rc1) - 2020-01-07
+* BREAKING
```

Comment:
Why are they all breaking changes?

Reply:
I think some of them are not real break. i.e. #8696 fixed a API bug. but the behavior changed. Maybe we should remove break label from them. @lafriks #8696 seems compitable with prvious implementation.

Reply:
Backported should not be there at all

Reply:
I think #8596 #8982 #8483 are not breaking

Reply:
@lafriks  #8596  is breaking! -> if you have a custom template  you have to change it!

Reply:
I commented #8982, it will break few workflow by default but they just need to change the value to false.

Reply:
OKay. Just removed break label of #8696 and keep others.

Reply:
Looks like #8596 was changed, maybe mistaken for #8696. 😅